test/utils/nrosched/nrosched.go

```diff
@@ -117,8 +119,12 @@ func CheckNROSchedulerAvailable(cli client.Client, NUMAResourcesSchedObjName str
 	return nroSchedObj
 }
 
-func IsSchedulingErrorCannotAlign(msg string) bool {
-	return strings.Contains(msg, ErrorCannotAlignPod)
+func IsSchedulingErrorCannotAlign(msg, policy string) bool {
```

Comment:
let's inline this code in `IsSchedulingErrorCannotAlign` - in hindsight, this helper should never have been extracted (let alone made public) in the first place

Reply:
Done.